components/Fields/ContribFieldGroup.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+/* eslint-disable vue/prop-name-casing */
+import { flow } from 'lodash'
+import type { ContribFields, Link } from '~/middleware/contrib-mode.global'
+import ExternalLink from '~/components/UI/ExternalLink.vue'
+
+const props = defineProps<{
+  editor_id: ContribFields['editor_id']
+  mapillary_link: ContribFields['mapillary_link']
+  osm_note: ContribFields['osm_note']
+}>()
+
+const onlyDefinedProps = computed(() => {
+  return flow([
+    Object.entries,
+    arr => arr.filter(([_key, value]: [key: string, value: Link ]) => !!value),
+    Object.fromEntries,
```

Comment:
Not just `_.omitBy` ?

Reply:
Yes I think it could be enough :)

Reply:
But I will use `pickBy` as it returns the exact object I need.